webapp/src/selectors/index.ts

```diff
@@ -13,21 +13,14 @@ const getPluginState = (state) => state['plugins-' + PluginId] || {};
 
 export const getPluginServerRoute = (state) => {
     const config = getConfig(state);
-    let basePath = '';
-    let url;
+    let baseUrl = config?.configSiteURL ? config.SiteUrl : window.location.href;
+    let basePath = new URL(baseUrl).pathname;
```

Comment:
```suggestion
    const baseUrl = new URL(config?.SiteURL ? config.SiteUrl : window.location.href);
    let basePath = baseUrl.pathname;
```

Reply:
I think it should be `const baseUrl = new URL(config?.SiteURL ? config.SiteURL : window.location.href);`

Reply:
How about
```
const baseUrl = new URL(config?.SiteURL ?? window.location.href);
```

Reply:
Its even better